File: HolaMundoJava/app/src/main/java/com/example/holamundojava/backend/OperacionesBasicas.java
package com.example.holamundojava.backend;

public class OperacionesBasicas {

    public OperacionesBasicas(){

    }
    public int suma(int a, int b){
        return a + b;
    }
    public int resta(int a, int b){
        return a - b;
    }
    public int multiplicacion(int a, int b){
        return a * b;
    }
    public String division(int a, int b){
        if(b == 0)
            return "No se puede dividir por cero";
        else
            return String.valueOf((double)a / b);
    }
}

File: HolaMundoKotlin/app/src/main/java/com/example/holamundokotlin/backend/OperacionesBasicas.kt
package com.example.holamundokotlin.backend


class OperacionesBasicas {
    fun suma(a: Int, b: Int): Int {
        return a + b
    }

    fun resta(a: Int, b: Int): Int {
        return a - b
    }

    fun multiplicacion(a: Int, b: Int): Int {
        return a * b
    }

    fun division(a: Int, b: Int): String {
        return if (b == 0) "No se puede dividir por cero" else (a.toDouble() / b).toString()
    }
}